packages/salesforcedx-utils/test/jest/i18n/message.test.ts

```diff
@@ -0,0 +1,93 @@
+import { Message, MISSING_LABEL_MSG } from '../../../src';
+
+describe('Message Unit Tests.', () => {
+  const messageBundle = {
+    a: 'a',
+    b: 'b',
+    c: 'c',
+    e: '%s:%s',
+    f: '1 %s 2 %s 3 %s'
+  };
+  const backupMessageBundle = { a: '1', b: '2', c: '3', d: '4' };
+  let delegate: Message;
+
+  beforeEach(() => {
+    delegate = new Message(backupMessageBundle);
+  });
+
+  it('Should be able to create an instance.', () => {
+    const message = new Message(messageBundle, delegate);
+    expect(message).toBeInstanceOf(Message);
+  });
+
+  it('Should find the label in the primary messageBundle.', () => {
+    const message = new Message(messageBundle, delegate);
+    const label = message.localize('a');
+    expect(label).toEqual(messageBundle.a);
+  });
+
+  it('Should find the label in the delegate messageBundle.', () => {
+    const message = new Message(messageBundle, delegate);
+    const label = message.localize('d');
+    expect(label).toEqual(backupMessageBundle.d);
+  });
+
+  describe('label not found', () => {
+    it('Should return error label if not found in either bundle.', () => {
+      const message = new Message(messageBundle, delegate);
+      const labelKey = 'notAKey';
+      const label = message.localize(labelKey);
+      const expected = `${MISSING_LABEL_MSG} ${labelKey}`;
+      expect(label).toEqual(expected);
+    });
+
+    it('Should return error label with args if not found in either bundle.', () => {
+      const message = new Message(messageBundle, delegate);
+      const labelKey = 'notAKey';
+      const arg1 = 'arg1';
+      const arg2 = 'arg2';
+      const label = message.localize(labelKey, arg1, arg2);
+      const expected = `${MISSING_LABEL_MSG} ${labelKey} (${arg1}) (${arg2})`;
+      expect(label).toEqual(expected);
+    });
+
+    it('Should return error label if no delegate and no lable found.', () => {
+      const message = new Message(messageBundle);
+      const labelKey = 'notAKey';
+      const label = message.localize(labelKey);
+      const expected = `${MISSING_LABEL_MSG} ${labelKey}`;
+      expect(label).toEqual(expected);
```

Comment:
It seems like these tests execute a `console.warn` that ends up being reported when `npm run test:unit `is run. Is that correct? Could something be stubbed so that this `console.warn` does not get output when the tests are run?
<img width="831" alt="2022-10-12_17-04-33" src="https://user-images.githubusercontent.com/46458081/195470114-5c7c8e9a-bacc-48a2-aaa5-860cfa4e912d.png">

Reply:
that is correct...I'll see what I can do